docs/guides/measure-qubits.ipynb

```diff
@@ -188,7 +195,70 @@
     "\n",
     "qc = QuantumCircuit(3, 1)\n",
     "qc.x([0, 2])\n",
-    "qc.measure_active()  # Measure qubits that are not idle, i.e., qubits 0 and 2."
+    "qc.measure_active()  # Measure qubits that are not idle, that is, qubits 0 and 2."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b1c5f875-6dfb-48e9-83f4-b25ba516af83",
+   "metadata": {},
+   "source": [
+    "### `MidCircuitMeasure` method\n",
+    "\n",
+    "\n",
+    "\n",
+    "Use `MidCircuitMeasure` (**NEED LINK**) to apply a mid-circuit measurement.  While you can use `QuantumCircuit.measure` for a mid-circuit measurement, `MidCircuitMeasure` is designed specifically for mid-circuit measurements, and is therefore typically a better choice.  For example, it adds less overhead to your circuit than when using QuantumCircuit.measure`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "86b25b9d-403c-40bd-b028-e65bb71b4ea4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit import QuantumCircuit\n",
+    "from qiskit_ibm_runtime.circuit import MidCircuitMeasure\n",
+    "from qiskit.circuit import Measure\n",
+    "\n",
+    "circ = QuantumCircuit(2, 2)\n",
+    "circ.x(0)\n",
+    "circ.append(MidCircuitMeasure(), [0], [0])\n",
+    "circ.append(MidCircuitMeasure(\"measure_3\"), [0], [1])\n",
```

Comment:
Let's not mention `measure_3` since it's not a thing (yet). This code would fail because no backend supports `measure_3`.